clients/unshaded/src/main/java/tachyon/client/file/options/SetStateOptions.java

```diff
@@ -73,4 +87,14 @@ private SetStateOptions(SetStateOptions.Builder builder) {
   public Boolean getPinned() {
     return mPinned;
   }
+
+  /**
+   * @return the ttl value, null if it is not set in builder; it specifies how long should the file
```

Comment:
if you would like to change the description, update the CreateOptions#getTTL description as well